### PR TITLE
feat: added Home page and Card component

### PR DESCRIPTION
**Pull request by krgarima**
## Change Logs:
# Description: Added Home Page and card component

1.Home Page - 
- This has cover image and 3 categories. On clicking `Play now` button of the Card, user is directed to the rules page of that category.
- There are 3 categories for now:
a. Cartoons
b. Anime
c. Animals

2. Card component -
- It is taking data from database `categories`  and displaying individual cards. 
Note: Card codes set here will be sent to the quiz page. On the quiz page, these card codes will be used to fetch data of that particular category.

3. Site has been made responsive.
Note: Navbar and Footer will be added later.

# Screenshots:
homeQUIZ1
homeQUIZ2

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,27 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { Routes, Route } from "react-router-dom";
+import Home from "./Pages/Home/Home";
+import Quiz from "./Pages/Quiz/Quiz";
+import Result from "./Pages/Result/Result";
+import Rules from "./Pages/Rules/Rules";
+import NavBar from "./Components/Navbar/Navbar";
+import Footer from "./Components/Footer/Footer";
```
Comment (vikrantkode) [inline src/App.js]: export all in one `index.js` file and import from there.

File: src/Pages/Home/Home.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <div className="home-container">
+      <h1 className="welcome-heading">QuizzWizz</h1>
+
+      <div className="img-container">
+        <img
+          className="cover-img"
+          src="/assets/images/bg5.jpg"
+          alt="cover"
+          srcSet=""
+        />
```
Comment (vikrantkode) [inline src/Pages/Home/Home.js]: `srcSet` is declared but no source given. Remove if not using this.

**Review by vikrantkode — APPROVED** [review]
- Few changes suggested for better experience, have a look on them
- Rest UI  and folder structure is good.

File: src/Components/Card/Card.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  justify-content: space-evenly;
+  width: 80vw;
+  flex-wrap: wrap;
+}
+
+.card {
+  min-height: 40vh;
+  width: 15.8rem;
+  border: 4px solid black;
+  background-color: white;
```
Comment (sakshi006) [inline src/Components/Card/Card.css:12]: use CSS variables for assigning colors

File: src/Components/Card/Card.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+export default function Card() {
+  const { setCategory } = useContext(CategoryContext);
+  return (
+    <div className="card-container">
+      {categories.map((cat) => (
+        <div key={Math.random()} className="card">
+          <div className="background-Image">
+            <img
+              className="card-img"
+              src="/assets/images/bg1.jpg"
```
Comment (sakshi006) [inline src/Components/Card/Card.js:16]: You can import images first and then directly apply them to the src attribute. [refer this](https://stackoverflow.com/questions/43823289/how-to-import-image-svg-png-in-a-react-component)

**Review by sakshi006 — APPROVED** [review]
Made few comments.
Looks good to merge

File: src/Components/Card/Card.css (hunk 3 above)
Comment (krgarima) [reply src/Components/Card/Card.css:12]: I will definitely do that later.

File: src/Components/Card/Card.js (hunk 4 above)
Comment (krgarima) [reply src/Components/Card/Card.js:16]: I tried but I was facing some issues so I removed it.

**Pull request merged**